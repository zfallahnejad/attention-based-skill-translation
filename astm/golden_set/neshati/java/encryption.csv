589259,encryption
474189,encryption
3474,encryption
238704,encryption
22656,encryption
1816580,encryption
149808,encryption
571407,encryption
2970947,encryption
372643,encryption
827060,encryption
