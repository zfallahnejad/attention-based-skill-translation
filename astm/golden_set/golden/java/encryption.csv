589259,144,176,0.45
474189,43,35,0.5512820512820513
3474,39,37,0.5131578947368421
238704,27,40,0.40298507462686567
22656,26,19,0.5777777777777777
1816580,19,11,0.6333333333333333
149808,14,18,0.4375
571407,12,17,0.41379310344827586
827060,11,8,0.5789473684210527
2970947,11,2,0.8461538461538461
372643,11,7,0.6111111111111112
